cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -140,6 +140,9 @@ func createKubeConfigFiles(outDir string, cfg *kubeadmapi.InitConfiguration, kub
 // NB. this method holds the information about how kubeadm creates kubeconfig files.
 func getKubeConfigSpecs(cfg *kubeadmapi.InitConfiguration) (map[string]*kubeConfigSpec, error) {
 	caCert, caKey, err := pkiutil.TryLoadCertAndKeyFromDisk(cfg.CertificatesDir, kubeadmconstants.CACertAndKeyBaseName)
+	if os.IsNotExist(errors.Cause(err)) {
+		return nil, errors.Wrap(err, "please run phase certs/ca to generate CA cert and key first")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "the CA files do not exist, please run `kubeadm init phase certs ca` subcommands to generate it.")
```

Just a suggestion. 
/cc @neolit123

Reply:
should we use the phase Depends / Dependencies string slice of the Phase object that @SataQiu  added?
/cc @SataQiu

Reply:
no, I explained here: https://github.com/kubernetes/kubernetes/pull/112941#issuecomment-1272498856.

Reply:
it's possible to run `kubeadm init phase certs ca` and then `kubeadm init phase kubeconfig/*`, go with dependency will fail in this case since it asks for run those phases in one cycle.

Reply:
If the phases can be executed independently, I recommend not overcoupling them now.

BTW, how about change "please run..." to "you can run `kubeadm init phase certs ca` to generate one"?

Reply:
thanks @SataQiu ! have updated according to the feedback from @pacoxu